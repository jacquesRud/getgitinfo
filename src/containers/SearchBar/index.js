import { FormControl } from "react-bootstrap";
export default function SearchBar({ setSearchedUser }) {
  return (
    <FormControl
      onBlur={(e) => {
        setSearchedUser(e.target.value);
      }}
    />
  );
}
