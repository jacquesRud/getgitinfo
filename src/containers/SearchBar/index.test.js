import { render } from "@testing-library/react";
import SearchBar from "./";

describe("SearchBar", () => {
  test("Render", () => {
    const { container } = render(<SearchBar setSearchedUser={null} />);
    expect(container).toMatchSnapshot();
  });
});
