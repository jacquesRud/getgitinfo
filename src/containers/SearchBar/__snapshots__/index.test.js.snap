// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Render 1`] = `
<div>
  <input
    class="form-control"
  />
</div>
`;
